src/sentry/static/sentry/app/components/group/externalIssueActions.jsx

```diff
@@ -170,10 +177,13 @@ class ExternalIssueActions extends AsyncComponent {
 
   constructor(props, context) {
     super(props, context);
-    this.state.showModal = false;
-    this.state.selectedIntegration = this.props.integration;
-    this.state.action = 'create';
-    this.state.issue = this.getIssue();
+
+    this.state = {
```

Comment:
This is going to overwrite state in `AsyncComponent` :/

Should probably override `getDefaultState`

Reply:
Ah doy

Reply:
That means these state vars won't reset when the component remounts btw

Reply:
That's the same behavior it had before if I understand correctly.

Reply:
It is, but it's possible it was overlooked before